app/assets/javascripts/components/login_modal.es6.jsx

```diff
@@ -87,9 +88,18 @@ class LoginModal extends React.Component {
   }
 
   _renderLoginModal() {
+    let $display = null
+    if (this.props.from_module) {
+      $display = (<div className={this.state.btnStyle} onClick={this._openModal}>
+          <i className="fa fa-download fa-lg"></i>
+          <span>Preview</span>
+          </div>)
+    } else {
+      $display = (<button className={this.state.btnStyle} onClick={this._openModal}>Log In</button>)
```

Comment:
The button on the resource module should still say "Download", but should open the login modal when people try to click it - just so they know that downloading is an option, but they need to log in to do so

![screen shot 2017-04-14 at 1 08 55 am](https://cloud.githubusercontent.com/assets/10657390/25037633/13c40574-20af-11e7-99f6-a8b7cca854b1.png)